ts/packages/anchor/src/provider.ts

```diff
@@ -17,6 +18,12 @@ import {
   SuccessfulTxSimulationResponse,
 } from "./utils/rpc.js";
 
+export function isVersionedTransaction(
+  transaction: Transaction | VersionedTransaction
+): transaction is VersionedTransaction {
+  return "version" in transaction;
```

Comment:
what is in transaction, is there another way? im guessing this is like a dict?

Reply:
this is taken from  the solana web3 js library